app/common/history.ts

```diff
@@ -0,0 +1,49 @@
+export interface UpdateStyleAction {
+  type: "update-style";
+  targets: Array<{ webviewId: string; selector: string }>;
+  selector: string;
+  style: string;
+  value: string;
+  originalValue: string;
+}
+
+export type Action = UpdateStyleAction;
```

Comment:
Nice I like this, leaves room for growth. We may eventually handle:
1. arbitrary props like src, className, etc. 
2. text 
3. structure (insert element, move element, remove element)